.claude/commands/audit-code.md

```
@@ -0,0 +1,165 @@
+# Audit Code Against Knowledge Rules
+
+Automated workflow to check and fix code based on knowledge rules and latest platform APIs.
+
+## Workflow
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)